docs/UserGuide.md

```diff
@@ -450,9 +585,23 @@ Format: `exit`
 
 #### 4.3 Saving the data :
 
-All tCheck data (i.e. contact details, ingredient data, sales data) are saved in the hard disk automatically after any
- command that changes the data. There is no need to save manually by entering any command.
+All tCheck data (i.e. ingredient data, sales data, employees' data) will be saved in the hard disk as json
+ file automatically after each command executes. There is no need to save manually by entering any command.
+ 
+ **:information_source: Notes:**<br>
+ * User should not attempt to make any change in all data files.<br>
+
+#### 4.4 Calendar :
+
+tCheck also provides a calendar pane when you start the application. It will automatically jump to the current today's
+ date based on your computer's date. It is designed to help you, a store manager, to quick access 
+ to the current month's calendar, so that you can easily find out what day is a specific date during 
+ your work planning.
```

Comment:
The sentence doesn't sound fluent. Perhaps this could be better phrased? The last half of the sentence is grammatically incorrect.

Reply:
Sure